libs/chain-signatures/contract/src/lib.rs

```diff
@@ -707,7 +707,7 @@ impl VersionedMpcContract {
             protocol_state: ProtocolContractState::Running(RunningContractState::new(
                 domains, keyset, parameters,
             )),
-            pending_requests: LookupMap::new(StorageKey::PendingRequests),
+            pending_requests: LookupMap::new(StorageKey::PendingRequestsV2),
             proposed_updates: ProposedUpdates::default(),
```

Comment:
What happens to the data under the old storage keys? Will it be persisted forever? Will it be deleted?
If we have any old updates, what will happen to them?

Reply:
Persisted forever. But we already had that happening in the past anyway. We can just run manual clearing before the upgrade, it's not a big deal - nobody cares.

Reply:
I don't know about old updates, that I think needs a more careful look at.

Reply:
OK, I filed #318 for that; it indeed is a problem.